protos/location/v1/location.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package protos.location.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "protos/location/v1";
+
+service LocationService {
+  rpc UpdateLocation(UpdateLocationRequest) returns (UpdateLocationResponse) {}
+  rpc QueryLocations(QueryLocationsRequest) returns (QueryLocationsResponse) {}
+}
+
+message UpdateLocationRequest {
+  string principal_id = 1;
+  string longitude = 2;
+  string latitude = 3;
+}
+
+message UpdateLocationResponse {
+}
+
+message QueryLocationsRequest {
+  string longitude = 1;
+  string latitude = 2;
+}
+
+message QueryLocationsResponse {
+  repeated QueryLocation locations = 1;
+}
+
+message QueryLocation {
+  string principal_id = 1;
+  float longitude = 2;
+  float latitude = 3;
```

Comment:
We might as well use `double` (64 bits) instead of `float` (32 bits).

* Proto docs, [Scalar Value Types](https://developers.google.com/protocol-buffers/docs/proto3)
* Google's [latlng.proto](https://github.com/googleapis/googleapis/blob/master/google/type/latlng.proto)